src/main/java/com/cloudbees/jenkins/plugins/BitbucketJobProbe.java

```diff
@@ -85,10 +84,32 @@ private boolean match(SCM scm, URIish url) {
                         return true;
                 }
             }
+        } else if (scm instanceof MercurialSCM) {
+            try {
+                URI hgUri = new URI(((MercurialSCM) scm).getSource());
+                String remote = url.toString();
+                if (almostMatches(hgUri, remote))
```

Comment:
Please, after an `if` open/close brackets `{`, `}` even if there is only one line.